controllers/chat/getSignedChannelList.js

```diff
@@ -1,7 +1,7 @@
 const {StreamChat} = require('stream-chat');
 const {responseSuccess} = require('../../utils/Responses');
 const {CHANNEL_TYPE} = require('../../helpers/constants');
-const {Topics} = require('../../databases/models');
+const {sequelize} = require('../../databases/models');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `sequelize` from `../../databases/models` has been replaced with the previous import of `Topics`. This change could potentially affect other parts of the code that use `Topics`. Ensure that this change does not break any functionality that relies on `Topics`.
```diff
- const {Topics} = require('../../databases/models');
+ const {sequelize} = require('../../databases/models');
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->